aioredisqueue/task.py

```diff
@@ -12,12 +15,32 @@ def generate_ack_info():
         return int(timeit.default_timer() * 1000)
 
     @staticmethod
-    def generate_id(payload=None):
-        return str(uuid.uuid4())
+    def generate_id(serialization_method):
+        return str(uuid.uuid4()) + '_' + serialization_method
```

Comment:
I'm not sure if we want `serialization_method` in task_id when we have homogeneous queue

Reply:
It appears to be a free flexible option and it doesn't make things worse, so why not?

Reply:
I think it is just a matter of taste. Avoiding data duplication seems to be the right thing to do

Reply:
Yes, there is some data duplication, but I decided to leave it as tradeoff to maximal flexibility and interoperability of homogenous and heterogenous queues

Reply:
So do we have solution for that? If it is decided to postpone it there should be a link to corresponding issue posted in this thread.

Reply:
I don't understand why did "the matter of taste" become something that must be fixed anyway. I explained why did I decide to leave this duplication, what's the problem?

Reply:
Well, we just don't have a consensus here. It doesn't mean that PR wouldn't be merged but if someone comes across the same ideas they wouldn'tfind this discussion.So if you are not going to change this, we need the discussion to continue in separate issue (which doesn't mean that it "must be fixed anyway".)

Reply:
Moved this discussion to #13